src/main/java/training/Robot.java

```diff
@@ -4,10 +4,17 @@ public class Robot {
 
     // Enter your subsystems...
 
-    public Robot(){
-        // Boot your subsystems...
+    public enum RobotType {
+        SIM,
```

Comment:
```suggestion
        SIMULATION,
```

Reply:
bro u wrote that 😭

Reply:
I’if Indian hacker hacks to your code base I expect you to fix it